internal/conf/load_test.go

```diff
@@ -0,0 +1,122 @@
+// INTECH Process Automation Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conf
+
+import (
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestEnv(t *testing.T) {
+
+	key := "KUIPER__BASIC__CONSOLELOG"
+	value := "true"
+
+	err := os.Setenv(key, value)
+	if err != nil {
+		t.Error(err)
+	}
+
+	c := KuiperConf{}
+	err = LoadConfig(&c)
+	if err != nil {
+		t.Error(err)
+	}
+
+	if c.Basic.ConsoleLog != true {
+		t.Errorf("env variable should set it to true")
+	}
+}
+
+func TestJsonCamelCase(t *testing.T) {
+	key := "KUIPER__DEFAULT__BODYTYPE"
```

Comment:
Here the key prefix should be the same as the yaml name. HTTPPULL_DEFAULT_BODYTYPE

Reply:
I have pushed fixes